### PR TITLE
Front end layout

### LINKED ISSUE #2: Figure Out Layout
Finding a popup model template for our flagship matching service.

Setting up the grid to display pets.

### PR DESCRIPTION
**Pull request by ainneo**
Hey guys, this is the initial setup for the frontend. There is still a lot of work to do, take a look through all the files... I created multiple components, some pages, a fake api to test data with... I am still need to work on a form, about page, create links... etc

### CODE REVIEW

File: destiny-app/src/components/hooks/fetchMain.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// import axios from 'axios';
```
Comment (JacobK233811) [inline destiny-app/src/components/hooks/fetchMain.js:1]: Why is all of this commented out for now?

Comment (ainneo) [reply destiny-app/src/components/hooks/fetchMain.js:1]: I am testing code to make things work. You will see some code that has been commented out, incase I need to use it later. Also some code is there for debugging and testing...

**Review by gideontong — CHANGES_REQUESTED** [review]
Please remove `.DS_Store` and `destiny-app/.eslintcache` from commit. The rest looks good.

File: destiny-app/src/components/hooks/fetchMain.js (hunk 1 above)
Comment (JacobK233811) [reply destiny-app/src/components/hooks/fetchMain.js:1]: Sounds good. What will axios do once un-commented? (Just curious, don't let it hold up the merge)

Comment (gideontong) [reply destiny-app/src/components/hooks/fetchMain.js:1]: It's the most popular HTTP client for Node.js, it's like requests for Python.

Comment (JacobK233811) [reply destiny-app/src/components/hooks/fetchMain.js:1]: Got it, thanks.

**Comment by JacobK233811** [comment]
Changes made.

**Pull request merged**